susemanager/src/mgr_bootstrap_data.py

```diff
@@ -1050,260 +488,248 @@
     },
     "SLE-15-x86_64": {
         "PDID": [1576, 2056, 1712],
         "BETAPDID": [1928],
-        "PKGLIST": ONLYSLE15 + PKGLIST15_SALT + PKGLIST15_X86_ARM,
+        "PKGLIST": PKGLIST15_SALT + PKGLIST15_X86_ARM,
         "DEST": DOCUMENT_ROOT + "/pub/repositories/sle/15/0/bootstrap/",
     },
     "SLES4SAP-15-ppc64le": {
         "PDID": [1588, 1613, 1710],
         "BETAPDID": [1926],
-        "PKGLIST": ONLYSLE15 + PKGLIST15_SALT + PKGLIST15_PPC,
+        "PKGLIST": PKGLIST15_SALT + PKGLIST15_PPC,
         "DEST": DOCUMENT_ROOT + "/pub/repositories/sle/15/0/bootstrap/",
     },
     "SLES4SAP-15-x86_64": {
         "PDID": [1576, 1612, 1712],
         "BETAPDID": [1928],
-        "PKGLIST": ONLYSLE15 + PKGLIST15_SALT + PKGLIST15_X86_ARM,
+        "PKGLIST": PKGLIST15_SALT + PKGLIST15_X86_ARM,
         "DEST": DOCUMENT_ROOT + "/pub/repositories/sle/15/0/bootstrap/",
     },
     "SLE-15-SP1-aarch64": {
         "PDID": [1769, 1709, 2216],
         "BETAPDID": [1925],
-        "PKGLIST": ONLYSLE15 + PKGLIST15_SALT + PKGLIST15_X86_ARM,
+        "PKGLIST": PKGLIST15_SALT + PKGLIST15_X86_ARM,
         "DEST": DOCUMENT_ROOT + "/pub/repositories/sle/15/1/bootstrap/",
     },
     "SLE-15-SP1-ppc64le": {
         "PDID": [1770, 1710, 2217],
         "BETAPDID": [1926],
-        "PKGLIST": ONLYSLE15 + PKGLIST15_SALT + PKGLIST15_PPC,
+        "PKGLIST": PKGLIST15_SALT + PKGLIST15_PPC,
         "DEST": DOCUMENT_ROOT + "/pub/repositories/sle/15/1/bootstrap/",
     },
     "SLE-15-SP1-s390x": {
         "PDID": [1771, 1711, 2218],
         "BETAPDID": [1927],
-        "PKGLIST": ONLYSLE15 + PKGLIST15_SALT + PKGLIST15_Z,
+        "PKGLIST": PKGLIST15_SALT + PKGLIST15_Z,
         "DEST": DOCUMENT_ROOT + "/pub/repositories/sle/15/1/bootstrap/",
     },
     "SLE-15-SP1-x86_64": {
         "PDID": [1772, 1712, 2219],
         "BETAPDID": [1928],
-        "PKGLIST": ONLYSLE15 + PKGLIST15_SALT + PKGLIST15_X86_ARM,
+        "PKGLIST": PKGLIST15_SALT + PKGLIST15_X86_ARM,
         "DEST": DOCUMENT_ROOT + "/pub/repositories/sle/15/1/bootstrap/",
     },
     "SUMA-40-PROXY-x86_64": {
         "PDID": [1772, 1908],
         "BETAPDID": [],
-        "PKGLIST": ONLYSLE15 + PKGLIST15_SALT + PKGLIST15_X86_ARM,
+        "PKGLIST": PKGLIST15_SALT + PKGLIST15_X86_ARM,
         "DEST": DOCUMENT_ROOT + "/pub/repositories/sle/15/1/bootstrap/",
     },
     "SLES4SAP-15-SP1-x86_64": {
         "PDID": [1772, 1712, 1766],
         "BETAPDID": [1928],
-        "PKGLIST": ONLYSLE15 + PKGLIST15_SALT + PKGLIST15_X86_ARM,
+        "PKGLIST": PKGLIST15_SALT + PKGLIST15_X86_ARM,
         "DEST": DOCUMENT_ROOT + "/pub/repositories/sle/15/1/bootstrap/",
     },
     "SLES4SAP-15-SP1-ppc64le": {
         "PDID": [1770, 1710, 1765],
         "BETAPDID": [1926],
-        "PKGLIST": ONLYSLE15 + PKGLIST15_SALT + PKGLIST15_PPC,
+        "PKGLIST": PKGLIST15_SALT + PKGLIST15_PPC,
         "DEST": DOCUMENT_ROOT + "/pub/repositories/sle/15/1/bootstrap/",
     },
```

Comment:
do we still need 15 SP0 and SP1? to my knowledge, they should  be  gone

Reply:
Miguel said that as long as "SLES 15" is supported we should keep all SPs.
That's also the reason why we still have all SLES12 SPs in our list